PredictIO-iOS/Classes/PIOZone.h

```diff
@@ -0,0 +1,34 @@
+//
+//  PIOZone.h
+//  PredictIOSDK
+//
+//  Created by Umer on 9/6/16.
+//  Copyright © 2016 ParkTAG GmbH. All rights reserved.
+//  Version 3.2.0
```

Comment:
Copyright text should be "Copyright (c) 2016 predict.io by ParkTAG GmbH. All rights reserved."
date format should be 06/09/2016
version text should be "SDK version 3.2.0"